packages/flutter/lib/src/widgets/async.dart

```diff
@@ -333,19 +343,25 @@ typedef Widget AsyncWidgetBuilder<T>(BuildContext context, AsyncSnapshot<T> snap
 class StreamBuilder<T> extends StreamBuilderBase<T, AsyncSnapshot<T>> {
   /// Creates a new [StreamBuilder] that builds itself based on the latest
   /// snapshot of interaction with the specified [stream] and whose build
-  /// strategy is given by [builder].
+  /// strategy is given by [builder]. The [initialData] is used to create the
+  /// first snapshot delivered to the [builder].
```

Comment:
This sentence is incorrect. While `initialData` controls the initial value of `_snapshot`, that value is only given to `builder`, if the `build` method is invoked by the Flutter framework prior to the receipt of the first stream event.